pkl-commons-cli/src/main/kotlin/org/pkl/commons/cli/commands/BaseOptions.kt

```diff
@@ -135,6 +136,14 @@ class BaseOptions : OptionGroup() {
       .path()
       .multiple()
 
+  // hidden option used by native tests
+  private val testPort: Int by
+    option(names = arrayOf("--test-port"), help = "Internal test option", hidden = true)
+      .single()
+      .int()
+      .default(-1)
+      .validate { it in 1024..65535 }
```

Comment:
Doesn't seem very useful to validate given it's a test flag, and it's also provisioned by the OS. Do we _know_ it's going to be in this range?

```suggestion
```

Reply:
Not terribly important to have this validated, but ephemeral ports are guaranteed to be high ports (1024-65535).
https://en.wikipedia.org/wiki/Ephemeral_port#Range

Reply:
Gotcha, thanks for the link. Let's still remove this, though; it adds some unnecessary brittleness to our tests.